confgenerator/testdata/bms/builtin/linux/builtin/golden/b4a0dead382dce7b4fe011d3f59fdb6d.lua

```diff
@@ -0,0 +1,17 @@
+
+function parser_nest(tag, timestamp, record)
+  local nestedRecord = {}
+  local parseKey = "message"
+  for k, v in pairs(record) do
+      if k ~= parseKey then
+          nestedRecord[k] = v
+      end
+  end
+
+  local result = {}
+  result[parseKey] = record[parseKey]
```

Comment:
This assumes "message" is in record. Might be good to wrap it around in an error handling clause

Reply:
I didn't write these lua files. They were auto-generated by "go test -mod=mod github.com/GoogleCloudPlatform/ops-agent/confgenerator -update -v" command. They appear to be identical to the ones generated for GCE instances. Considering this, I think it would be more appropriate to address your concerns within the component responsible for their generation (I don't know what component it is though).

Reply:
I believe it's [here](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/fluentbit/parser_deduplication.go#L33). Luckily, this Lua bit is verbatim in the code, so it should be easy to send a PR.